src/development/accessibility-and-localization/accessibility.md

```diff
@@ -144,7 +144,34 @@ widgets to offer a dramatically more accessible experience.
 
 <iframe width="560" height="315" src="{{site.youtube-site}}/embed/bWbBgbmAdQs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## Testing accessibility on web:
+## Testing accessibility on mobile
+
+You can use Flutter's <a href="https://api.flutter.dev/flutter/flutter_test/AccessibilityGuideline-class.html">Accessibility Guideline API</a> to test if your mobile app's UI meets the accessibility recommendations for text contrast, target size, and target labels. 
+
+The example below shows how to use the Guideline API on <a href="https://docs.flutter.dev/get-started/codelab/?tab=talkback">Startup Name Generator</a>, the app built as part of the codelab on creating your first Flutter app. Each list tile on the app's main screen is tappable target with text represented in 18 point. You can add the tests provided by the Guideline API in `test/widget_test.dart` of your app directory.
```

Comment:
**issues:** 
- Replace "below" as it runs counter to [Google's documentation accessibility guidelines](https://developers.google.com/style/accessibility).
- Use [semantic line breaks](https://sembr.org/) for copy.

```suggestion
The following example shows how to use the Guideline API on
<a href="https://docs.flutter.dev/get-started/codelab/?tab=talkback">Startup Name Generator</a>,
the app built as part of the codelab on creating your first Flutter app.
Each list tile on the app's main screen serves as a tappable target
with text represented in 18 point. You can add Guideline API tests
in `test/widget_test.dart` of your app directory.
```